AirCasting/Assets.xcassets/ABLoading.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "Group 2.png",
+      "idiom" : "universal",
+      "scale" : "1x"
```

Comment:
I don't think we support 1x scale devices

Reply:
True, I removed the 1x scale and added a task to Trello to review other images and remove 1x scaled images & add missing scales. Thanks!